share/spice/cryptocurrency/cryptocurrencylist.txt

```diff
@@ -531,6 +531,7 @@ sxc,sexcoin,sex coin,
 sha,shacoin,sha coin,
 shade,shadecoin,shade coin,
 sdc,shadowcoin,shadow coin,
+sdc, shadowcash, shadow cash
```

Comment:
@kewde I think it should be fine as it will use `sdc` to check the API which is the same for both shadowcoin and shadowcash. Thanks!